validate-date.js

```diff
@@ -1,31 +1,50 @@
-function validateDate(dateValue) {
-    if (dateValue == null) {
-        return 'Invalid Format';
-    }
-    return dateValidator(dateValue);
+function validateDate(dateValue, boolean = false) {
+  let responses = boolean
+    ? [false, false, false, true]
+    : ["Invalid Format", "Invalid Date", "Invalid Format", "Valid Date"];
+  if (dateValue == null) {
+    return responses[0];
+  }
+  return dateValidator(dateValue, responses);
 }
 
 function daysInMonth(year, month) {
-    const days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
-    return (month === 2 && year % 4 === 0) ? 29 : days[month - 1];
+  const days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+  return month === 2 && year % 4 === 0 ? 29 : days[month - 1];
```

Comment:
I am used to two spaced indentation, but I think that's a personal choice. Looks cleaner either way :)

Also, let's put back the "common braces ( )"? Although it has no significance technically I think it aids in readability. Thoughts?

Reply:
The space indentation was automatically added by Prettier in VS Code, sorry for not paying attention to how it was (it is so easy to just press crtl+i and not care about it).
I will re-add the parenthesis - I have never thought about that before but it does improve readability.

Reply:
This was automatic identantion by Prettier...I have no preferences whatsoever

Reply:
I get that.. totally. I have been in your shoes too :)

Reply:
I get that.. totally. I have been in your shoes too :)

Reply:
>Also, let's put back the "common braces ( )"? Although it has no significance technically I think it aids in readability. 

I see this is not yet addressed in your recent changes. It'd be great if you can add that back. Thanks :)